isis/tests/FunctionalTestsIsisImport.cpp

```diff
@@ -0,0 +1,226 @@
+#include <iostream>
+#include <time.h>
+
+#include <QRegExp>
+#include <QString>
+#include <nlohmann/json.hpp>
+
+#include "Fixtures.h"
+#include "md5wrapper.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "PvlKeyword.h"
+
+#include "isisimport.h"
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+using json = nlohmann::json;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/isisimport.xml").expanded();
+
+TEST_F(TempTestingFiles, FunctionalTestIsisImportLabelXmlInput) {
+  QString labelFileName = "data/isisimport/pds4.xml";
```

Comment:
Is this file also missing?

Reply:
The tests write out the xml label to go along side the pds4.img file in the test data